tests/pipelines/test_pipelines_common.py

```diff
@@ -1347,7 +1347,7 @@ def test_components_function(self):
 
     @unittest.skipIf(torch_device not in ["cuda", "xpu"], reason="float16 requires CUDA or XPU")
     @require_accelerator
-    def test_float16_inference(self, expected_max_diff=5e-2):
+    def test_float16_inference(self, expected_max_diff=6e-2):
```

Comment:
`6e-2` vs `5e-2` is ok, what's the context of this change though?

Reply:
sorry, it's an un-deleted experimental change, will change back in next PR. Sorry for unnecessary confusion